code401challenges/src/main/java/code401challenges/linkedList/LinkedList.java

```diff
@@ -141,4 +140,47 @@ public int getNthFromEnd(int k) {
         return -1;
     }
 
+    public static LinkedList mergeLists(LinkedList one, LinkedList two) {
+        if (one.head == null) return two;
+        if (two.head == null) return one;
+
+        boolean checkerListOne = false;
+        boolean checkerListTwo = false;
+
+        LinkedList result = new LinkedList();
+
+        while (one.head != null && two.head != null) {
+
+            result.append(one.head.data);
```

Comment:
How come you're using `append()`? It has an O(n) time complexity versus O(1) on your insert.